joss/paper.md

```diff
@@ -30,40 +30,40 @@ Consequently, these populations undergo evolution [@pennock2007models], and can
+This information reveals the sequences of events behind gain, loss, or maintenance of specific traits, and also facilitates making inferences about the underlying mechanisms driving a given evolutionary system [mooers1997inferring;@dolsonInterpretingTapeLife2020;@moreno2023toward].
 
 The Phylotrack project provides libraries for tracking and analyzing phylogenies in *in silico* evolution.
 The project is composed of 1) Phylotracklib: a header-only C++ library, developed under the umbrella of the Empirical project [@ofria2020empirical], and 2) Phylotrackpy: a Python wrapper around Phylotracklib, created with Pybind11 [@pybind11].
 Both components supply a public-facing API to attach phylogenetic tracking to digital evolution systems, as well as a stand-alone interface for measuring a variety of popular phylogenetic topology metrics [@tuckerGuidePhylogeneticMetrics2017].
-The underlying algorithm design prioritizes efficiency, allowing Phylotrack to support large systems with rapid generational turnover.
-The underlying C++ implementation ensures fast, memory-efficient performance, with multiple explicit features (e.g., phylogeny pruning and abstraction, etc.) that facilitate reducing the memory footprint of phylogenetic information.
+The underlying algorithm design prioritizes efficiency, allowing Phylotrack to support large agent populations with rapid generational turnover.
+The underlying C++ implementation ensures fast, memory-efficient performance, with multiple explicit features (e.g., phylogeny pruning and abstraction, etc.) that can reduce the memory footprint of phylogenetic information.
 
 # Statement of Need
 
 <!-- was going to say a rich history of sophisticated phylogenetic analyses... -->
 *In silico* evolution work enjoys a rich history of phylogenetic record-keeping and analysis, and many systems facilitate tracking phylogenies [@ray1992evolution;@ofria2004avida;@bohm2017mabe;@de2012deap;@Garwood_REvoSim_Organism-level_simulation_2019].
 However, to our knowledge, no other general-purpose perfect phylogeny tracking library exists; prior work has used bespoke system- or framework-specific implementations.
-In contrast, Phylotrack provides ready-built tracking flexible enough to attach to any system containing replicating entities.
+In contrast, Phylotrack provides ready-built tracking flexible enough to attach to any simulation of replicating entities.
 
-Two other general-purpose libraries for recording phylogenetic information do exist: hstrat and Automated Phylogeny Over Geological Timescales (APOGeT).
-However, they allow different modes of phylogenetic tracking than Phylotrack does.
+Two other general-purpose libraries for phylogenetic tracking do exist: hstrat and Automated Phylogeny Over Geological Timescales (APOGeT).
```

Comment:
This is nit-picky, but I feel like hstrat and APOGeT aren't exactly tracking phylogenies - they are doing something more sophisticated. Struggling to think of a word that captures that though.

Reply:
Agreed, I was half thinking the same thing when I wrote it. We used record-keeping in the previous sentence I think

phylogenetic journaling?

Reply:
What if we make the previous one "phylogenetic measurement" and use "record-keeping" here?

Reply:
Making the previous one "phylogenetic instrumentation" might more clearly differentiate from analysis and kind of get at that idea of the record-keeping system.

Reply:
but either is fine with me I'll happily hand over the judgement call on that

Reply:
Ooo, I really like "phylogenetic instrumentation" but I'm a little worried it will be opaque to readers

Reply:
I think it's the best term to use in the next sentence though. Adding it.